libs/wallet/src/vega-transaction-dialog/vega-transaction-dialog.tsx

```diff
@@ -17,6 +19,8 @@ export interface VegaTransactionDialogProps {
   transaction: VegaTxState;
   reset: () => void;
   title?: string;
+  edit?: (body?: unknown) => Promise<unknown>;
```

Comment:
question (blocking): Why is the body unknown here?

Reply:
updated to be Order type